.github/workflows/ci.yaml

```diff
@@ -37,3 +37,11 @@ jobs:
       shell: bash
       run: |
         cmake --workflow --preset ${{matrix.target.preset}}
+
+    - name: Upload vcpkg error logs
+      if: failure()
+      uses: actions/upload-artifact@v4
+      with:
+        name: logs-${{matrix.target.preset}}
+        path: |
+          /usr/local/share/vcpkg/**/*.log
```

Comment:
Not necessary for this PR, but adds a nice way to debug issues in CI.
Can be removed and put up as a separate PR.